src/repos/UserRepo.ts

```diff
@@ -16,17 +18,18 @@ const createUser = async (
 
   // if no user, create a new one
   if (!users[0]) {
```

Comment:
how does this not give an error? what if it's an empty list.. shouldn't we be checking for length

Reply:
In javascript it returns undefined in that case, which is falsy